src/helics/core/BrokerBase.cpp

```diff
@@ -150,6 +151,24 @@ std::shared_ptr<helicsCLI11App> BrokerBase::generateBaseCLI()
         "--globaltime",
         globalTime,
         "specify that the broker should use a globalTime coordinator to coordinate a master clock time with all federates");
+    hApp->add_flag(
+        "--async",
```

Comment:
```suggestion
        "--asynctime",
```

Matches the `globaltime` option by adding a `time` suffix. Might also make it clearer that the option is related to time coordination and not something else like communication.